docs/docs/zh/guide/task/stored-procedure.md

```diff
@@ -1,12 +1,47 @@
 # 存储过程节点
 
+## 综述
+
+`PROCEDURE` 任务类型，用于执行存储过程的程序
+
+## 创建任务
+
 - 根据选择的数据源，执行存储过程。
 > 拖动工具栏中的![PNG](https://analysys.github.io/easyscheduler_docs_cn/images/toolbar_PROCEDURE.png)任务节点到画板中，如下图所示：
```

Comment:
Please change this png link from `analysys.github.io` to our local file like you add then in its `Task Example` section